frontend/src/routes/+page.svelte

```diff
@@ -3,6 +3,22 @@
   import TgHeadingMain from "$lib/components/base/TgHeadingMain.svelte";
   import TgHeadingTitle from "$lib/components/base/TgHeadingTitle.svelte";
   import TgParagraph from "$lib/components/base/TgParagraph.svelte";
+  import TgLink from "$lib/components/base/TgLink.svelte";
+  import { get } from "$lib/services/rest";
+  import { onMount } from "svelte";
+
+  var status = true;
```

Comment:
This should use `let` instead of `var`, and boolean variables should generally be prefixed with `is` to be more descriptive.
```suggestion
  let isApiAvailable = true;
```